packages/cms-lib/modules.js

```diff
@@ -50,7 +116,7 @@ async function validateSrcAndDestPaths(src, dest) {
   }
   // src is a .module folder but dest is not
   // e.g. `upload foo.module bar`
-  if (isModuleFolder(absoluteSrc) && !isModuleFolder(dest)) {
+  if (isModuleFolder(_src) && !isModuleFolder(_dest)) {
```

Comment:
Not introduced in this PR but I think that the error message could be a bit better. Something like `The destination path "${dest}" is not a valid module path`.